setup.cfg

```diff
@@ -41,6 +42,8 @@ warn_redundant_casts = True
 warn_unused_ignores = True
 disallow_any_generics = True
 check_untyped_defs = True
+# Change to `plugins = pydantic.mypy:strict` to type-check and remove `**kwargs` from BaseModel.__init__:
+plugins = pydantic.mypy
```

Comment:
> Very dumb/simple question: how is this plugin distributed? As part of pydantic or as a separate package?

I wasn't sure  of the answer, so I tried this (based on the mypy docs) and it just worked. If you change `pydantic.mypy` to `pydantic.mypy:strict`, it will fail the external mypy test (since the external test involves inputs getting parsed, so aren't the right types in the input to the `__init__`); this is my proof it is being used :)

Reply:
:+1: